internal/sdkv2provider/resource_cloudflare_workers_domain_test.go

```diff
@@ -0,0 +1,127 @@
+package sdkv2provider
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+)
+
+
+const (
+	scriptContent = `addEventListener('fetch', event => {event.respondWith(new Response('test'))});`
+)
+func TestAccCloudflareWorkerDomain_Attach(t *testing.T) {
+	var domain cloudflare.WorkersDomain
+	rnd := generateRandomResourceName()
+	hostname := rnd + ".com"
```

Comment:
we have a domain you can use via env vars `CLOUDFLARE_DOMAIN` that you can prefix a subdomain with a random set of characters